Specification/4._CAN_bus_transport_layer.md

```diff
@@ -19,43 +19,39 @@ UAVCAN defines the following types of transfers:
 Both message and service transfers can be further distinguished between:
 
 * Single-frame transfer - a transfer that is entirely contained in a single CAN frame.
-* Multi-frame transfer - a transfer whose payload is distributed over multiple CAN frames.
+* Multi-frame transfer - a transfer that has its payload distributed over multiple CAN frames.
 
 The following properties are common to all types of transfer:
 
 Property        | Description
 ----------------|------------------------------------------------------------------------------------------------------
 Payload         | The serialized data structure
-Data type ID    | This identifier indicates how the data structure should be interpreted
-Priority        | See below
-Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from others
+Data type ID    | An identifier that indicates how the data structure should be interpreted
+Priority        | An positive integer value that defines the message urgency (higher numbers indicate reduced priority). Higher priority transfers can delay transmission of loser priority transfers. 
+Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from all others
 
-The transfer priority is an integer number that defines the urgency of the data contained in the transfer.
-Numerically lower priority values indicate higher urgency, and numerically higher values indicate lower urgency.
-Transfers of higher priority (i.e. with numerically lower priority value) can delay transmission of transfers
-whose priority are lower (i.e. numerically higher).
 
 ### Message broadcasting
 
-A broadcasted message is carried by a single message transfer that contains the serialized message.
-A broadcasted message has the following properties:
+A broadcast message is carried by a single-frame transfer that contains the serialized message.
```

Comment:
@pavel-kirienko Check this one - did you mean single-frame as I have changed?

Reply:
Sorry, no. Perhaps I worded it clumsily here which caused confusion. What I meant is that a broadcasted message is always carried by a single transfer, i.e. it can't be split over multiple transfers.

Is "broadcasted" not a valid form, or is it merely not recommended? I'm asking in order to avoid similar mistakes in the future. Wiktionary says that the "-ed" form is in use, but doesn't say whether it is legal: https://en.wiktionary.org/wiki/broadcasted.

Reply:
> Sorry, no. Perhaps I worded it clumsily here which caused confusion. What I meant is that a broadcasted message is always carried by a single transfer, i.e. it can't be split over multiple transfers.

The confusion here is basically again "what is a transfer". It never occurred to me that if a transfer is the act of emitting a message, that you might be able to split a message over multiple transfers (if you see what I mean).  Lets see how you respond to my transfer question above. For now I have reverted this.

Reply:
> Is "broadcasted" not a valid form, or is it merely not recommended? I'm asking in order to avoid similar mistakes in the future. Wiktionary says that the "-ed" form is in use, but doesn't say whether it is legal: https://en.wiktionary.org/wiki/broadcasted.

Broadcasted is "OK" to refer to the past tense - "A message that was broadcasted" but in this case I think we're using a future tense: "a message to be broadcast". Unfortunately I can't find a definitive guide on this, but my understanding is that broadcast is always *preferred*.